lib/fa2/asset/multi_asset.impl.jsligo

```diff
@@ -0,0 +1,263 @@
+#import "../common/errors.mligo" "Errors"
+
+#import "../common/tzip12.datatypes.jsligo" "TZIP12"
+
+#import "../common/tzip12.interfaces.jsligo" "TZIP12Interface"
+
+#import "../common/tzip16.datatypes.jsligo" "TZIP16"
+
+export namespace MultiAsset implements TZIP12Interface.FA2{
+   export type Ledger = big_map<[address, nat], nat>;
```

Comment:
Why this type start by an uppercase ? It should be ledger no ? It's not a namespace

Reply:
yes, I will rename all of this like it is on mligo. It was a jsligo namespace before refactoring